### PR TITLE
Parse/fix!/no longer support escaping triple quote

### PR DESCRIPTION
**Pull request by Huy-DNA**
## Summary
* BREAKING CHANGE: No longer support escaping triple quote

## Issue
(issue link here)

## Lasting Changes (Technical)

(please list down: code changes/things that have wide-effect; new libraries/functions added that can be used by others; examples below)

* (Added `class EmailValidator` to validate email address' validity)
* (Added `Tenant#is_trial?` check)

## Checklist

Please check directly on the box once each of these are done

- [x] Documentation (if necessary)
- [X] Tests (integration test/unit test)
- [ ] Integration Tests Passed
- [ ] Code Review

### CODE REVIEW

File: packages/dbml-parse/tests/lexer/input/unclosed_strings.in.dbml [hunk 1]
```diff
@@ -4,6 +4,4 @@
 
 '''This is also an unclosed string
 
-\''''''
```
Comment (NQPhuc) [inline packages/dbml-parse/tests/lexer/input/unclosed_strings.in.dbml:7]: Please help add at least one more test for case of `'''... \''''` to show that it work

**Review by NQPhuc — COMMENTED** [review]
To account for this change please also help:
* Update the dbml syntax docs
* Update the dbml exporter to:
  * Remove escape of triple quotes
  * Always escape single quotes

File: dbml-homepage/docs/docs.md [hunk 2]
```diff
@@ -454,7 +454,8 @@ Note: '''
 - Line continuation: `\` backslash
 - Escaping characters:
   - `\`: using double backslash `\\`
-  - `'''`: using `\'''`
+  - `'`: using `\'`
+  - `"`: using `\"`
```
Comment (NQPhuc) [inline dbml-homepage/docs/docs.md:458]: I don't remember we ever have this rule, it this behavior newly introduced in this PR or we just didn't document them before?

Comment (Huy-DNA) [reply dbml-homepage/docs/docs.md:458]: we didn't document this before anh, I think because we explicitly escape `'`, we should mention this rule.

**Review by NQPhuc — APPROVED** [review]
LGTM!

**Pull request merged**